### PR TITLE
OTP-888 Swagger including mobility profiles.

### PR DESCRIPTION
**Pull request by JymDyerIBI**
### Checklist

- [ ] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [ ] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [ ] The description lists all applicable issues this PR seeks to resolve
- [ ] The description lists any configuration setting(s) that differ from the default settings
- [ ] All tests and CI builds passing
- [ ] The description lists all relevant PRs included in this release _(remove this if not merging to master)_

### Description
Add mobility profile support. New fields in `OtpUser` (in this code and in Mongo and Swagger). Following the logic, naming, and strings in the Georgia Tech document, a somewhat unwieldy new `OtpUserController#calculateMobilityMode()` that takes specified string values and calculates a "mobility mode" keyword or compound keyword.

This PR should include the https://github.com/ibi-group/otp-middleware/commit/08d96959576cb3e902e837d368e5a23d05b37a82 commits.

### CODE REVIEW

File: src/main/resources/latest-spark-swagger-output.yaml [hunk 1]
```diff
@@ -2550,8 +2550,18 @@ definitions:
         type: "boolean"
       hasConsentedToTerms:
         type: "boolean"
+      isMobilityLimited:
```
Comment (binh-dam-ibigroup) [inline src/main/resources/latest-spark-swagger-output.yaml:2553]: Main reason why you want to publish a PR branch quickly for review: I see that the mobility attributes start to stack up, and my thought was to put them under a `mobilityProfile` field:
```
- <other OTP User attributes>
- mobilityProfile
    - isMobiityLimited
    - mobilityDevices
    - mobilityMode  
    - visionLimitation
```

**Review by binh-dam-ibigroup — APPROVED** [review]
Approving this to make tests pass.

**Review by miles-grant-ibigroup — APPROVED** [review]
Happy to see this updated. Do unit test need updating?

File: src/main/resources/latest-spark-swagger-output.yaml (hunk 1 above)
Comment (JymDyerIBI) [reply src/main/resources/latest-spark-swagger-output.yaml:2553]: I'm putting together a `mobilityProfile` field as requested.

**Pull request merged**